framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceJavaParserStorage.java

```diff
@@ -757,23 +757,28 @@ private void addClass(
               @Nullable @BinaryName String classNameKey,
               @Nullable TypeDeclaration<?> javaParserNode) {
             String className;
```

Comment:
The following changes seem to be an improvement that is independent of the rest of your changes?  How about making a separate pull request with them?  Then, you can pull master into your branch, which will make this pull request's diffs smaller.

Reply:
I reverted these changes and created a new pull request to address this issue.
https://github.com/typetools/checker-framework/pull/5914